### ISSUE TITLE
request

### ISSUE THREAD

**Issue by PieflavorPiePiePie** [BitCrackers/AmongUsMenu#532]
pull requests are just.. comparing files.
so this is the only place i can do a request.
basically, in among us when the host uses hacks/other things they can make it 0 kill cooldown.
so actually 2 requests
1: adding 0 kill cooldown would be cool if it's possible
2: make it so you can type in any number for all of the settings.
well any number that among us won't realize you're hacking with anyways
and also any number that will make it so it doesn't crash
basically: cheat without being caught and cheat without making it crash

**Comment by william1east** [comment]
Support from the main team was stopped on 12/29/2022, if you do want these changes the project is open source.
